pkg/kor/all.go

```diff
@@ -198,6 +207,8 @@ func GetUnusedAll(filterOpts *filters.Options, clientset kubernetes.Interface, a
 		allDiffs = append(allDiffs, namespaceStatefulsetDiff)
 		namespaceRoleDiff := getUnusedRoles(clientset, namespace, filterOpts)
 		allDiffs = append(allDiffs, namespaceRoleDiff)
+		namespaceClusterRoleDiff := getUnusedClusterRoles(clientset, namespace, filterOpts)
```

Comment:
Cluster Roles are not namespaced.
Please move this to where the crds and PV's are.
At this point we should consider moving it out to a function(all 3 of these non namespaced objects)
See the comment on the behavior of the search for unused